src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -327,20 +365,29 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             // Add all listable functions details to the payload
             JObject functions = new JObject();
-            string routePrefix = await WebFunctionsManager.GetRoutePrefix(hostOptions.RootScriptPath);
+            // TODO move before L387 and remove Console.WriteLine
+            JToken extensionsPayload = GetExtensionsPayload();
+            Console.WriteLine($"Extensions Payload\n{extensionsPayload.ToString()}");
+            // TODO move before L395 and remove Console.WriteLine
+            JToken concurrencyPayload = _concurrencyOptionProvider.GetConcurrencyOption();
+            Console.WriteLine($"Concurrency Payload\n{concurrencyPayload.ToString()}");
+
             var listableFunctions = _functionMetadataManager.GetFunctionMetadata().Where(m => !m.IsCodeless());
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
             // TEMP: refactor this code to properly add extensions in all scenario(#7394)
             // Add the host.json extensions to the payload
-            if (_environment.IsKubernetesManagedHosting())
+
```

Comment:
The TEMP comment above can be removed as part of this PR right, since that's what we're addressing

Reply:
Sure! Thanks! Will do.

Reply:
Done